fvm/evm/types/account.go

```diff
@@ -0,0 +1,47 @@
+package types
+
+// Account is an EVM account, currently
+// three types of accounts are supported on Flow EVM,
+// externally owned accounts (EOAs), smart contract accounts and flow owned accounts (FOAs)
```

Comment:
According to the PR description should the name be changed from FOA to bridged account?

Reply:
good catch, updated